components/x-timeline-feed/src/TimelineFeed.jsx

```diff
@@ -0,0 +1,40 @@
+import { h } from '@financial-times/x-engine';
+import { Teaser, presets } from '@financial-times/x-teaser';
+import { getArticleGroups } from './lib/transform';
+import styles from './TimelineFeed.scss';
+import classNames from 'classnames';
+
+const TimelineFeed = props => {
+	const { articleCustomSlot = () => {} } = props;
+	const articleGroups = getArticleGroups(props);
+
+	return articleGroups.length && (
+		<div className={classNames(styles.root)}>
+			{articleGroups.map(group => (
+				<section key={group.date} className={classNames(styles.articleGroup)}>
+					<h2 className={classNames(styles.articleGroup__heading)}>{group.title}</h2>
+					<ul className={classNames(styles.articleGroup__articles)}>
+						{group.articles.map(article => {
+							const slotContent = typeof articleCustomSlot === 'function' ? articleCustomSlot(article): articleCustomSlot;
+
+							return (
+								<li key={article.id} className={styles.article}>
+									<Teaser
```

Comment:
As mentioned in #207 I think it may be better to not couple these components at all and provide the component to be rendered in this slot as a prop instead.

If it is better to keep the two components tightly coupled then perhaps this component should be called `x-teaser-list` instead to make that explicit.

Reply:
Talking to @benbarnett about this, the myFT feed timeline view is the only use case we have for this - we're x-dashing it so we can use it both the web and the app.

We also have a need for a simple list of teasers for use on the myFT Save Articles page - that won't have the time/date localisation and grouping that makes up the bulk of this component.

So our conclusion was it would be better to rename this component to `x-teaser-timeline`, and have a separate component (`x-teaser-list` probably) that may well have more use cases in future.

Reply:
PR to rename this to **x-teaser-timeline**: https://github.com/Financial-Times/x-dash/pull/219